includes/class-wcsatt-display.php

```diff
@@ -526,6 +536,43 @@ public static function filter_price_html( $price, $product ) {
 
 		return $price;
 	}
+
+	/**
+	 * Override the WooCommerce "Add to Cart" button text with "Sign Up Now".
+	 *
+	 * @since 1.1.1
+	 */
+	public static function add_to_cart_text( $button_text ) {
+		global $product;
+
+		$product_type = $product->product_type;
+
+		$product_schemes = get_post_meta( $product->id, '_wcsatt_schemes', true );
+
+		$force_subscription = get_post_meta( $product->id, '_wcsatt_force_subscription', true );
+
+		if ( in_array( $product_type, WCS_ATT()->get_supported_product_types() ) && $product_schemes ) {
+
+			if ( $force_subscription == 'yes' ) {
```

Comment:
@thenbrent I obviously still have much to learn. I did not know of this Yoda condition. I will push a quick fix.